eve/workers/openstack-multiple-nodes/terraform/network.tf

```diff
@@ -6,9 +8,62 @@ variable "openstack_network" {
   }
 }
 
+# Default internal networks
+variable "control_plane_cidr" {
+  type    = "string"
+  default = "172.21.254.0/28"
```

Comment:
Can I suggest to use *another* subnet for these two, such that we never accidentally make assumptions on the Vagrant environment being 'the default'?

Reply:
Yes we can, what would fit in your opinion? `172.22.254.X` instead of `172.21.254.X` would be OK?

Reply:
Done: 070c4a8